packages/framework/src/Component/Cron/CronFacade.php

```diff
@@ -130,6 +140,12 @@ protected function runModule(CronModuleExecutor $cronModuleExecutor, CronModuleC
             $this->cronModuleFacade->isModuleSuspended($cronModuleConfig)
         );
 
+        try {
+            $this->mailer->flushSpoolQueue();
+        } catch (Swift_TransportException $exception) {
+            $this->logger->error(sprintf('Exception occurred while flushing email queue: %s', $exception->getMessage()));
```

Comment:
See: vendor/symfony/symfony/src/Symfony/Component/Console/EventListener/ErrorListener.php:43 for better usage:
`return $this->logger->error('An error occurred while using the console. Message: "{message}"', ['exception' => $error, 'message' => $error->getMessage()]);`